PWGCF/DataModel/FemtoDerived.h

```diff
@@ -115,8 +115,8 @@ DECLARE_SOA_TABLE(FemtoDreamParticles, "AOD", "FEMTODREAMPARTS",
                   femtodreamparticle::PIDCut,
                   femtodreamparticle::TempFitVar,
                   femtodreamparticle::Indices,
-                  femtodreamparticle::MinvLambda,     // TEMPORARY
-                  femtodreamparticle::MinvAntiLambda, // TEMPORARY
+                  femtodreamparticle::MinvLambda,
+                  femtodreamparticle::MinvAntiLambda,
```

Comment:
The names are a bit misleading, `InvMLambda` and `InvMAntiLambda` perhaps?

Reply:
I adopted the same nomenclature used in the V0 tables